src/components/screens/dashboard/dashboard-screen.tsx

```diff
@@ -6,53 +6,58 @@ import { PersonTotalTime } from "../../../generated/client";
 import { errorAtom } from "../../../atoms/error";
 import LoaderWrapper from "../../generics/loader-wrapper";
 import { getHoursAndMinutes } from "../../../utils/time-utils";
+import strings from "../../../localization/strings";
 
 /**
  * Dashboard screen component
  */
-function DashboardScreen () {
+function DashboardScreen() {
```

Comment:
Components should be declared using arrow function syntax.

Reply:
Changed to arrow function